packages/vendure-plugin-sort-by-popularity/package.json

```diff
@@ -20,14 +20,18 @@
   },
   "devDependencies": {
     "@types/jest": "27.4.1",
-    "@vendure/ui-devkit": "1.9.2",
     "@vendure/admin-ui-plugin": "1.9.2",
     "@vendure/core": "1.9.2",
     "@vendure/testing": "1.9.2",
+    "@vendure/ui-devkit": "1.9.2",
     "jest": "27.2.0",
     "nodemon": "2.0.15",
     "ts-jest": "27.0.5",
     "ts-node": "10.7.0",
     "typescript": "4.3.5"
+  },
+  "dependencies": {
+    "@vendure/job-queue-plugin": "^1.9.5",
+    "bullmq": "1"
```

Comment:
These should be devdependencies, because a project consuming the plugin will have their own JobQueue setup.

Also, the worker workst without these right? Other plugins also use the worker successfully (and in tests also)

Reply:
I think I should remove that. I don't think we need it